Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -71,27 +71,25 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         "Should move location to trash": function () {
             var capiMock = new Mock(),
                 contentServiceMock = new Mock(),
+                discoveryServiceMock = new Mock(),
                 locationId = 'ronaldo-luis-nazario-de-lima',
-                loadRootResponse = {
-                    "document": {
-                        "Root": {
-                            "trash": {
-                                "_href": 'gianfranco-zola'
-                            }
-                        }
-                    }
-                };
+                getInfoObjectResponse = {"_href": 'fabrizio-ravanelli'};
 
             Mock.expect(capiMock, {
                 method: 'getContentService',
                 returns: contentServiceMock,
             });
 
-            Mock.expect(contentServiceMock, {
-                method: 'loadRoot',
-                args: [Mock.Value.Function],
-                run: function (cb) {
-                    cb(false, loadRootResponse);
+            Mock.expect(capiMock, {
+                method: 'getDiscoveryService',
+                returns: discoveryServiceMock,
+            });
+
+            Mock.expect(discoveryServiceMock, {
+                method: 'getInfoObject',
+                args: [Mock.Value.String, Mock.Value.Function],
```

Comment:
`Mock.Value.String` is not enough precise, I guess this should be `"Trash"` here